OrbitQt/orbitmainwindow.ui

```diff
@@ -241,22 +241,10 @@
     <addaction name="actionReport_Missing_Feature"/>
     <addaction name="actionReport_Bug"/>
    </widget>
-   <widget class="QMenu" name="menuCrash">
```

Comment:
When looking at this PR's commits one by one, it's unclear why these changes to the .ui-file are needed. Looks like an accident. Maybe you could add a sentence to the commit message?

Reply:
Did you get this from an intermediate commit? I added a hotkey to the "clear capture" button temporarily to reproduce the behavior, but all UI changes should have been reverted.

Reply:
Yes you revert them in the last commit. (before the fixup commit). But when looking at the first commit it's unclear why the whole crash-menu is deleted. I suspect this might also not compile on its own. Ideally commits should compile on its own to make bisecting possible.

Reply:
Link: https://github.com/google/orbit/pull/885/commits/1ee8cf2f56a1bfda1ec8f0145bb3c0958ead825f

Reply:
As discussed out-of-band: Since this will be squashed it's pointless to fix that. Thanks!

Reply:
As discussed in GVC, not really needed when we squash/merge - but fixed anyways, thanks for showing me how!